app/templates/components/nav.twig

```diff
@@ -0,0 +1,46 @@
+<div style="z-index:1000;" class="container col-md-12 p-0 m-0  sticky-top">
+    <nav class="navbar navbar-expand-md navbar-light bg-light rounded-bottom">
+
+        <!-- <img src="/web-assets/images/BDPA-Flights-Black.jpeg" class="rounded" width="44" height="44" alt="" loading="lazy"> -->
+
+        <a href="<?=$_ENV['BASE_URL']?>" class="navbar-brand mb-0 h1 ml-3">Ghostmeme</a> <!-- Make name Airlanta -->
+        <div class="col col-md w-100 justify-content-center">
+            <form method='get' action="<?=$_ENV['BASE_URL']?>">
+                <input name='search' type='search' placeholder='search'>
+                <!-- <button class = 'btn btn-primary'type="submit"> Search </button> -->
+            </form>
+        </div>
+            <div class="navbar-nav ml-auto">
+            
+
+                <?php
```

Comment:
We should try NOT to have PHP code in templates- the idea is to separate the logic (code) from design (html). This session evaluation code could probably be moved into `init.php`, since it will be executed on almost every page (except login and signup).

The Sidebar can be another template in the "components" directory